frontend/src/metabase/auth/actions.ts

```diff
@@ -123,7 +124,10 @@ export const logout = createAsyncThunk(
         dispatch(clearCurrentUser());
         await dispatch(refreshLocale()).unwrap();
         trackLogout();
-        dispatch(push(Urls.login()));
+
+        // We use old react-router-redux which references old redux, which does not require
+        // action type to be a string - unlike RTK v2+
+        dispatch(push(Urls.login()) as unknown as UnknownAction);
```

Comment:
`push` from the old `react-router-redux` does not use the new `UnknownAction` type

https://redux-toolkit.js.org/usage/migrating-rtk-2#anyaction-deprecated-in-favour-of-unknownaction